storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/operations/get/GetObjectStub.java

```diff
@@ -0,0 +1,174 @@
+package com.pushtorefresh.storio.sqlite.operations.get;
+
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.pushtorefresh.storio.sqlite.Changes;
+import com.pushtorefresh.storio.sqlite.SQLiteTypeMapping;
+import com.pushtorefresh.storio.sqlite.StorIOSQLite;
+import com.pushtorefresh.storio.sqlite.queries.Query;
+import com.pushtorefresh.storio.sqlite.queries.RawQuery;
+
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+import rx.Observable;
+
+import static java.util.Collections.singleton;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+class GetObjectStub {
+
+    @NonNull
+    final StorIOSQLite storIOSQLite;
+
+    @NonNull
+    private final StorIOSQLite.Internal internal;
+
+    @NonNull
+    final Query query;
+
+    @NonNull
+    final RawQuery rawQuery;
+
+    @NonNull
+    final GetResolver<TestItem> getResolver;
+
+    @NonNull
+    private final Cursor cursor;
+
+    @NonNull
+    final TestItem item;
+
+    @NonNull
+    private final SQLiteTypeMapping<TestItem> typeMapping;
+
+    private final boolean withTypeMapping;
+
+    @SuppressWarnings("unchecked")
```

Comment:
Oh, sorry if you've copied this idea from our code.

Can you please remove this `@SuppressWarnings` and change them to comments from the IDE on the exact lines, `@SupressWarnings` hides too much.